cmd/fyne/internal/commands/build.go

```diff
@@ -443,5 +443,9 @@ func normaliseVersion(str string) string {
 	if pos := strings.Index(str, "-0.20"); pos != -1 {
 		str = str[:pos] + "-dev"
 	}
+	
```

Comment:
Looks like there is a trailing tab character in here

Reply:
fixed